.gitignore

```diff
@@ -37,6 +37,3 @@ server/storage/
 .floobit
 .floo
 .flooignore
-
```

Comment:
oops, I deleted that line by accident when I deleted my own .config git ignore for my API key..
to piggy-backed it (and my data file) to your tempsecret.js and /store.